### PR TITLE
Kevin/quiz mode

### PR DESCRIPTION
**Pull request by matthewcoufal**
- Quiz-mode
  - Make quiz mode overlay on poll builder
  - Update MC to include correct answer (refer to zeplin designs)

### CODE REVIEW

File: Clicker/Models/Parsers/PollParser.swift [hunk 1]
```diff
@@ -17,6 +17,10 @@ class PollParser: Parser {
         let id = json[ParserKeys.idKey].intValue
         let text = json[ParserKeys.textKey].stringValue
         let results = json[ParserKeys.resultsKey].dictionaryValue
+        results.forEach { (key, value) in
```
Comment (AAAstorga) [inline Clicker/Models/Parsers/PollParser.swift]: is this still needed or can it be removed now?

File: Clicker/Views/Cards/MCResultCell.swift [hunk 2]
```diff
@@ -25,7 +25,7 @@ class MCResultCell: UICollectionViewCell {
     var percentSelected: Float!
     var highlightViewWidthConstraint: Constraint!
     var didLayoutConstraints: Bool = false
-    var missedCorrectAnswer: Bool = false
+    var showCorrectAnswer: Bool = false
```
Comment (AAAstorga) [inline Clicker/Views/Cards/MCResultCell.swift]: no need to declare this a Bool. It will be inferred

File: Clicker/Views/Cards/MCResultCell.swift [hunk 3]
```diff
@@ -160,21 +160,18 @@ class MCResultCell: UICollectionViewCell {
             if let correctAnswer = correctAnswer {
                 if correctAnswer != "" {
```
Comment (AAAstorga) [inline Clicker/Views/Cards/MCResultCell.swift]:
```suggestion
                if !correctAnswer.isEmpty {
```

File: Clicker/Views/Cards/MCResultCell.swift (hunk 2 above)
Comment (matthewcoufal) [reply Clicker/Views/Cards/MCResultCell.swift]: I'll take that out. Thanks!

File: Clicker/Models/Parsers/PollParser.swift (hunk 1 above)
Comment (matthewcoufal) [reply Clicker/Models/Parsers/PollParser.swift]: Not needed anymore, so I'll remove that. Thanks!

**Pull request merged**